spack/packages/xios/gcc_remap.patch

```diff
@@ -1,9 +1,26 @@
---- a/extern/remap/src/meshutil.cpp	(revision 2397)
-+++ b/extern/remap/src/meshutil.cpp	(revision 2398)
-@@ -5,4 +5,5 @@
- #include "earcut.hpp"
- #include <vector>
+Index: extern/remap/src/earcut.hpp
+===================================================================
+--- a/extern/remap/src/earcut.hpp       (revision 2252)
++++ b/extern/remap/src/earcut.hpp       (working copy)
+@@ -6,6 +6,9 @@
+#include <memory>
+#include <vector>
+#include <limits>
++// Using cstdint ensures that the symbols are in the
++// global namespace and in std ... which is how uint32_t is used
++#include <cstdint>
+//#include <tuple>
+//#include <cstdint> 
+//#include <cstddef>
```

Comment:
Can these please be cleaned up (remember to adjust the patch line numbers to suit)

Reply:
Not sure what you mean here. The `//#include` are part of the existing code (not part of this patch). Or do you mean the comments I added (about using cstdint vs stdint.h)?

Reply:
Ah my mistake sorry, thought these were added lines